transports/janus_mqtt.c

```diff
@@ -377,6 +401,43 @@ int janus_mqtt_init(janus_transport_callbacks *callback, const char *config_path
 		ctx->admin.publish.topic = NULL;
 	}
 
+	/* Status messages configuration */
+	janus_config_item *status_enabled_item = janus_config_get(config, config_status, janus_config_type_item, "enabled");
+	if(status_enabled_item && status_enabled_item->value && janus_is_true(status_enabled_item->value)) {
+		ctx->status.enabled = TRUE;
+
+		janus_config_item *status_connect_message_item = janus_config_get(config, config_status, janus_config_type_item, "connect_message");
+		if(status_connect_message_item && status_connect_message_item->value) {
+			ctx->status.connect_message = g_strdup(status_connect_message_item->value);
+		} else {
+			ctx->status.connect_message = NULL;
+		}
+
+		janus_config_item *status_disconnect_message_item = janus_config_get(config, config_status, janus_config_type_item, "disconnect_message");
+		if(status_disconnect_message_item && status_disconnect_message_item->value) {
+			ctx->status.disconnect_message = g_strdup(status_disconnect_message_item->value);
+		} else {
+			ctx->status.disconnect_message = NULL;
+		}
+
+		janus_config_item *status_topic_item = janus_config_get(config, config_status, janus_config_type_item, "topic");
+		if(status_topic_item && status_topic_item->value) {
+			ctx->status.topic = g_strdup(status_topic_item->value);
+		} else {
+			ctx->status.topic = g_strdup(JANUS_MQTT_DEFAULT_STATUS_TOPIC);
+		}
+
+		janus_config_item *status_qos_item = janus_config_get(config, config_status, janus_config_type_item, "qos");
+		if(status_qos_item && status_qos_item->value) {
+			ctx->status.qos = atoi(status_qos_item->value);
```

Comment:
What happens if you put a negative number here?

Reply:
```
[FATAL] [transports/janus_mqtt.c:janus_mqtt_init:465] Can't connect to MQTT broker, return code: -9
```